README.rst

```diff
@@ -1,9 +1,19 @@
-.. image:: https://travis-ci.org/MagicStack/uvloop.svg?branch=master
-    :target: https://travis-ci.org/MagicStack/uvloop
+
+.. image:: https://img.shields.io/pypi/v/uvloop.svg
+   :target: https://pypi.python.org/pypi/uvloop
 
 .. image:: https://img.shields.io/pypi/status/uvloop.svg?maxAge=2592000?style=plastic
     :target: https://pypi.python.org/pypi/uvloop
 
+.. image:: https://travis-ci.org/MagicStack/uvloop.svg?branch=master
+    :target: https://travis-ci.org/MagicStack/uvloop
+
+.. image:: https://img.shields.io/pypi/dm/uvloop.svg
+   :target: https://pypi.python.org/pypi/uvloop
+
+.. image:: https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat
+   :target: http://uvloop.readthedocs.org/en/latest/
+
 
```

Comment:
I don't like when READMEs have a lot of shields -- it's hard to see the really important ones.  Let's keep the current shields (2) as is, and provide a link to the docs site.